src/main/java/org/commcare/cases/instance/StorageBackedChildElement.java

```diff
@@ -198,6 +198,11 @@ public TreeReference getRef() {
         return ref;
     }
 
+    @Override
+    public void clearVolatiles() {
+        // we should also clear the other cache implementation in this class in future
```

Comment:
@ctsims Think it's for me to clear this cache now but I can't find a way to do it without using something like - 

````
for (int i = 0; i < getNumChildren(); i++) {
            getChildAt(i).clearVolatiles();
}
````

Reply:
nvm, It was simple.